integrationTests/interface.go

```diff
@@ -39,3 +50,42 @@ type NetworkShardingUpdater interface {
 	UpdatePeerIdShardId(pid core.PeerID, shardId uint32)
 	IsInterfaceNil() bool
 }
+type (
```

Comment:
add empty line before

Reply:
why is type "(" needed ? couldn't be type Facade interface { etc }

Reply:
Didn't knew this was even possible. Changing to align with the rest of the code.

Reply:
Done